e2e/iam/orgs_api_key_test.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build e2e iam
+
+package iam_test
+
+import (
+	"encoding/json"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/mongodb/mongocli/e2e"
+	"github.com/stretchr/testify/assert"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+const (
+	iamEntity     = "iam"
+	orgEntity     = "orgs"
+	projectEntity = "projects"
+	apiKeysEntity = "apikeys"
+)
+
+func TestOrgAPIKeys(t *testing.T) {
+	cliPath, err := e2e.Bin()
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	var ID string
+
+	// This test must run first to grab the ID of the org to later describe
+	t.Run("List", func(t *testing.T) {
+		cmd := exec.Command(cliPath, iamEntity, orgEntity, apiKeysEntity, "ls")
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+
+		if err != nil {
+			t.Fatalf("unexpected error: %v, resp: %v", err, string(resp))
+		}
+		var keys []mongodbatlas.APIKey
+		if err := json.Unmarshal(resp, &keys); err != nil {
+			t.Fatalf("unexpected error: %v", err)
+		}
+		assert.NotEmpty(t, keys)
+	})
```

Comment:
[ni] I'd run `List` after `Create` to check that we get 1

Reply:
we do, `assert.NotEmpty(t, keys)` already, there's always one, the one we use for the tests ;)

Reply:
I don't know why I read `assert.Empty`. Thanks